fdb-relational-core/src/main/antlr/RelationalParser.g4

```diff
@@ -645,7 +645,8 @@ helpStatement
 // details
 
 describeObjectClause
-    : (
+    : level=(VERBOSE | MINIMAL)?
```

Comment:
I don't think we want `MINIMAL` verbosity here. Please remove it and leave an optional `VERBOSE`.